internal/v2/s2av2.go

```diff
@@ -69,14 +71,24 @@ func (c *s2av2TransportCreds) ClientHandshake(ctx context.Context, serverAuthori
 	if err != nil {
 		return nil, nil, err
 	}
+	// Create an AccessTokenManager instance to use to authenticate to S2Av2.
+	os.Setenv("S2A_ACCESS_TOKEN", "token")
+	accessTokenManager, err := tokenmanager.NewSingleTokenAccessTokenManager()
```

Comment:
Can we store the accessTokenManager as a field in the s2av2TransportCreds struct, so that we're not creating a new one for each TLS connection?